scripts/plugins/alerts/security/retransmissions/user_scripts/flow/retransmissions.lua

```diff
@@ -0,0 +1,78 @@
+--
```

Comment:
There was already script `pro/scripts/pro_plugins/alerts/network/tcp_connection_issues/user_scripts/flow/tcp_issues_generic.lua`
doing similar things. Please, explain what is the difference.